### ISSUE TITLE
Rhel/Centos 6 Compilation error since yesterday

### ISSUE THREAD

**Issue by emphazer** [SpiderLabs/ModSecurity#1407]
Since yesterday it is not possible to compile the sources anymore with rhel6 httpd 2.2.15

````
Executing(%build): /bin/sh -e /var/tmp/rpm-tmp.cduVKL
+ umask 022
+ cd /builddir/build/BUILD
+ cd ModSecurity-master
+ LANG=C
+ export LANG
+ unset DISPLAY
+ ./autogen.sh
libtoolize: putting auxiliary files in AC_CONFIG_AUX_DIR, `build'.
libtoolize: copying file `build/ltmain.sh'
libtoolize: putting macros in AC_CONFIG_MACRO_DIR, `build'.
libtoolize: copying file `build/libtool.m4'
libtoolize: copying file `build/ltoptions.m4'
libtoolize: copying file `build/ltsugar.m4'
libtoolize: copying file `build/ltversion.m4'
libtoolize: copying file `build/lt~obsolete.m4'
configure.ac:21: installing `build/config.guess'
configure.ac:21: installing `build/config.sub'
configure.ac:17: installing `build/install-sh'
configure.ac:17: installing `build/missing'
alp2/Makefile.am: installing `build/depcomp'
+ CFLAGS='-O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector --param=ssp-buffer-size=4 -m64 -mtune=generic'
+ export CFLAGS
+ CXXFLAGS='-O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector --param=ssp-buffer-size=4 -m64 -mtune=generic'
+ export CXXFLAGS
+ FFLAGS='-O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector --param=ssp-buffer-size=4 -m64 -mtune=generic -I/usr/lib64/gfortran/modules'
+ export FFLAGS
+ ./configure --build=x86_64-redhat-linux-gnu --host=x86_64-redhat-linux-gnu --target=x86_64-redhat-linux-gnu --program-prefix= --prefix=/usr --exec-prefix=/usr --bindir=/usr/bin --sbindir=/usr/sbin --sysconfdir=/etc --datadir=/usr/share --includedir=/usr/include --libdir=/usr/lib64 --libexecdir=/usr/libexec --localstatedir=/var --sharedstatedir=/var/lib --mandir=/usr/share/man --infodir=/usr/share/info --enable-pcre-match-limit=no --enable-pcre-match-limit-recursion=no --enable-pcre-study --with-yajl=/usr/include/yajl --enable-lua-cache --with-curl
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... /bin/mkdir -p
checking for gawk... gawk
checking whether make sets $(MAKE)... yes
checking build system type... x86_64-redhat-linux-gnu
checking host system type... x86_64-redhat-linux-gnu
checking for style of include used by make... GNU
checking for x86_64-redhat-linux-gnu-gcc... no
checking for gcc... gcc
checking for C compiler default output file name... a.out
checking whether the C compiler works... yes
checking whether we are cross compiling... no
checking for suffix of executables... 
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether gcc accepts -g... yes
checking for gcc option to accept ISO C89... none needed
checking dependency style of gcc... gcc3
checking for a sed that does not truncate output... /bin/sed
checking for grep that handles long lines and -e... /bin/grep
checking for egrep... /bin/grep -E
checking for fgrep... /bin/grep -F
checking for ld used by gcc... /usr/bin/ld
checking if the linker (/usr/bin/ld) is GNU ld... yes
checking for BSD- or MS-compatible name lister (nm)... /usr/bin/nm -B
checking the name lister (/usr/bin/nm -B) interface... BSD nm
checking whether ln -s works... yes
checking the maximum length of command line arguments... 1572864
checking whether the shell understands some XSI constructs... yes
checking whether the shell understands "+="... yes
checking for /usr/bin/ld option to reload object files... -r
checking for x86_64-redhat-linux-gnu-objdump... no
checking for objdump... objdump
checking how to recognize dependent libraries... pass_all
checking for x86_64-redhat-linux-gnu-ar... no
checking for ar... ar
checking for x86_64-redhat-linux-gnu-strip... no
checking for strip... strip
checking for x86_64-redhat-linux-gnu-ranlib... no
checking for ranlib... ranlib
checking command to parse /usr/bin/nm -B output from gcc object... ok
checking how to run the C preprocessor... gcc -E
checking for ANSI C header files... yes
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
checking for dlfcn.h... yes
checking for objdir... .libs
checking if gcc supports -fno-rtti -fno-exceptions... no
checking for gcc option to produce PIC... -fPIC -DPIC
checking if gcc PIC flag -fPIC -DPIC works... yes
checking if gcc static flag -static works... no
checking if gcc supports -c -o file.o... yes
checking if gcc supports -c -o file.o... (cached) yes
checking whether the gcc linker (/usr/bin/ld -m elf_x86_64) supports shared libraries... yes
checking whether -lc should be explicitly linked in... no
checking dynamic linker characteristics... GNU/Linux ld.so
checking how to hardcode library paths into programs... immediate
checking for shl_load... no
checking for shl_load in -ldld... no
checking for dlopen... no
checking for dlopen in -ldl... yes
checking whether a program can dlopen itself... yes
checking whether a statically linked program can dlopen itself... yes
checking whether stripping libraries is possible... yes
checking if libtool supports shared libraries... yes
checking whether to build shared libraries... yes
checking whether to build static libraries... yes
checking for gawk... (cached) gawk
checking for x86_64-redhat-linux-gnu-gcc... gcc
checking whether we are using the GNU C compiler... (cached) yes
checking whether gcc accepts -g... (cached) yes
checking for gcc option to accept ISO C89... (cached) none needed
checking dependency style of gcc... (cached) gcc3
checking how to run the C preprocessor... gcc -E
checking for a BSD-compatible install... /usr/bin/install -c
checking whether ln -s works... yes
checking whether make sets $(MAKE)... (cached) yes
checking for grep that handles long lines and -e... (cached) /bin/grep
checking for perl... /usr/bin/perl
checking for env... /usr/bin/env
checking for ANSI C header files... (cached) yes
checking fcntl.h usability... yes
checking fcntl.h presence... yes
checking for fcntl.h... yes
checking limits.h usability... yes
checking limits.h presence... yes
checking for limits.h... yes
checking for stdlib.h... (cached) yes
checking for string.h... (cached) yes
checking for unistd.h... (cached) yes
checking for sys/types.h... (cached) yes
checking for sys/stat.h... (cached) yes
checking sys/utsname.h usability... yes
checking sys/utsname.h presence... yes
checking for sys/utsname.h... yes
checking for an ANSI C-conforming const... yes
checking for inline... inline
checking for C/C++ restrict keyword... __restrict
checking for pid_t... yes
checking for size_t... yes
checking whether struct tm is in sys/time.h or time.h... time.h
checking for uint8_t... yes
checking for stdlib.h... (cached) yes
checking for GNU libc compatible malloc... yes
checking for working memcmp... yes
checking for atexit... yes
checking for getcwd... yes
checking for memmove... yes
checking for memset... yes
checking for strcasecmp... yes
checking for strchr... yes
checking for strdup... yes
checking for strerror... yes
checking for strncasecmp... yes
checking for strrchr... yes
checking for strstr... yes
checking for strtol... yes
checking for fchmod... yes
checking for strcasestr... yes
checking host system type... (cached) x86_64-redhat-linux-gnu
Checking platform... Identified as Linux
checking for libcurl config script... no
configure: *** curl library not found.
configure: NOTE: curl library is only required for building mlogc
configure: NOTE: mlgoc compilation was disabled.
configure: looking for Apache module support via DSO through APXS
configure: found apxs at /usr/sbin/apxs
configure: checking httpd version
configure: httpd is recent enough
checking for libpcre config script... /usr/bin/pcre-config
configure: using pcre v7.8
checking for libapr config script... /usr/bin/apr-1-config
configure: using apr v1.3.9
checking for libapu config script... /usr/bin/apu-1-config
configure: using apu v1.3.9
checking for libxml2 config script... /usr/bin/xml2-config
checking if libxml2 is at least v2.6.29... yes, 2.7.6
configure: using libxml2 v2.7.6
checking for x86_64-redhat-linux-gnu-pkg-config... no
checking for pkg-config... /usr/bin/pkg-config
checking pkg-config is at least version 0.9.0... yes
checking for liblua config script... /usr/bin/pkg-config
configure: using lua v5.1.3
checking for libyajl config script... /usr/bin/pkg-config
configure: using yajl v2.1.1
checking for ssdeep path... no
configure: optional ssdeep library not found
configure: WARNING: APR util was not compiled with crypto support. SecRemoteRule will not support the parameter 'crypto'
configure: creating ./config.status
config.status: creating Makefile
config.status: creating tools/Makefile
config.status: creating apache2/Makefile
config.status: creating build/apxs-wrapper
config.status: creating tests/run-unit-tests.pl
config.status: creating tests/run-regression-tests.pl
config.status: creating tests/gen_rx-pm.pl
config.status: creating tests/csv_rx-pm.pl
config.status: creating tests/regression/server_root/conf/httpd.conf
config.status: creating tools/rules-updater.pl
config.status: creating tests/Makefile
config.status: creating apache2/modsecurity_config_auto.h
config.status: executing depfiles commands
config.status: executing libtool commands
+ sed -i 's|^hardcode_libdir_flag_spec=.*|hardcode_libdir_flag_spec=""|g' libtool
+ sed -i 's|^runpath_var=LD_RUN_PATH|runpath_var=DIE_RPATH_DIE|g' libtool
+ make -j16
Making all in tools
make[1]: Entering directory `/builddir/build/BUILD/ModSecurity-master/tools'
make[1]: Nothing to be done for `all'.
make[1]: Leaving directory `/builddir/build/BUILD/ModSecurity-master/tools'
Making all in apache2
make[1]: Entering directory `/builddir/build/BUILD/ModSecurity-master/apache2'
make  all-am
make[2]: Entering directory `/builddir/build/BUILD/ModSecurity-master/apache2'
/bin/sh ../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I.  -DLINUX=2 -D_REENTRANT -D_GNU_SOURCE -I/usr/include/libxml2   -I/usr/include/apr-1 -I/usr/include/apr-1 -I/usr/include/httpd -I/usr/include/libxml2 -DWITH_LUA -DWITH_PCRE_STUDY    -DREQUEST_EARLY  -DCACHE_LUA -DWITH_YAJL -I/usr/include/yajl  -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector --param=ssp-buffer-size=4 -m64 -mtune=generic  -I/usr/include/apr-1  -MT mod_security2_la-acmp.lo -MD -MP -MF .deps/mod_security2_la-acmp.Tpo -c -o mod_security2_la-acmp.lo `test -f 'acmp.c' || echo './'`acmp.c
/bin/sh ../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I.  -DLINUX=2 -D_REENTRANT -D_GNU_SOURCE -I/usr/include/libxml2   -I/usr/include/apr-1 -I/usr/include/apr-1 -I/usr/include/httpd -I/usr/include/libxml2 -DWITH_LUA -DWITH_PCRE_STUDY    -DREQUEST_EARLY  -DCACHE_LUA -DWITH_YAJL -I/usr/include/yajl  -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector --param=ssp-buffer-size=4 -m64 -mtune=generic  -I/usr/include/apr-1  -MT mod_security2_la-apache2_config.lo -MD -MP -MF .deps/mod_security2_la-apache2_config.Tpo -c -o mod_security2_la-apache2_config.lo `test -f 'apache2_config.c' || echo './'`apache2_config.c
/bin/sh ../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I.  -DLINUX=2 -D_REENTRANT -D_GNU_SOURCE -I/usr/include/libxml2   -I/usr/include/apr-1 -I/usr/include/apr-1 -I/usr/include/httpd -I/usr/include/libxml2 -DWITH_LUA -DWITH_PCRE_STUDY    -DREQUEST_EARLY  -DCACHE_LUA -DWITH_YAJL -I/usr/include/yajl  -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector --param=ssp-buffer-size=4 -m64 -mtune=generic  -I/usr/include/apr-1  -MT mod_security2_la-apache2_io.lo -MD -MP -MF .deps/mod_security2_la-apache2_io.Tpo -c -o mod_security2_la-apache2_io.lo `test -f 'apache2_io.c' || echo './'`apache2_io.c
/bin/sh ../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I.  -DLINUX=2 -D_REENTRANT -D_GNU_SOURCE -I/usr/include/libxml2   -I/usr/include/apr-1 -I/usr/include/apr-1 -I/usr/include/httpd -I/usr/include/libxml2 -DWITH_LUA -DWITH_PCRE_STUDY    -DREQUEST_EARLY  -DCACHE_LUA -DWITH_YAJL -I/usr/include/yajl  -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector --param=ssp-buffer-size=4 -m64 -mtune=generic  -I/usr/include/apr-1  -MT mod_security2_la-apache2_util.lo -MD -MP -MF .deps/mod_security2_la-apache2_util.Tpo -c -o mod_security2_la-apache2_util.lo `test -f 'apache2_util.c' || echo './'`apache2_util.c
/bin/sh ../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I.  -DLINUX=2 -D_REENTRANT -D_GNU_SOURCE -I/usr/include/libxml2   -I/usr/include/apr-1 -I/usr/include/apr-1 -I/usr/include/httpd -I/usr/include/libxml2 -DWITH_LUA -DWITH_PCRE_STUDY    -DREQUEST_EARLY  -DCACHE_LUA -DWITH_YAJL -I/usr/include/yajl  -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector --param=ssp-buffer-size=4 -m64 -mtune=generic  -I/usr/include/apr-1  -MT mod_security2_la-mod_security2.lo -MD -MP -MF .deps/mod_security2_la-mod_security2.Tpo -c -o mod_security2_la-mod_security2.lo `test -f 'mod_security2.c' || echo './'`mod_security2.c
/bin/sh ../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I.  -DLINUX=2 -D_REENTRANT -D_GNU_SOURCE -I/usr/include/libxml2   -I/usr/include/apr-1 -I/usr/include/apr-1 -I/usr/include/httpd -I/usr/include/libxml2 -DWITH_LUA -DWITH_PCRE_STUDY    -DREQUEST_EARLY  -DCACHE_LUA -DWITH_YAJL -I/usr/include/yajl  -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector --param=ssp-buffer-size=4 -m64 -mtune=generic  -I/usr/include/apr-1  -MT mod_security2_la-modsecurity.lo -MD -MP -MF .deps/mod_security2_la-modsecurity.Tpo -c -o mod_security2_la-modsecurity.lo `test -f 'modsecurity.c' || echo './'`modsecurity.c
/bin/sh ../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I.  -DLINUX=2 -D_REENTRANT -D_GNU_SOURCE -I/usr/include/libxml2   -I/usr/include/apr-1 -I/usr/include/apr-1 -I/usr/include/httpd -I/usr/include/libxml2 -DWITH_LUA -DWITH_PCRE_STUDY    -DREQUEST_EARLY  -DCACHE_LUA -DWITH_YAJL -I/usr/include/yajl  -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector --param=ssp-buffer-size=4 -m64 -mtune=generic  -I/usr/include/apr-1  -MT mod_security2_la-msc_status_engine.lo -MD -MP -MF .deps/mod_security2_la-msc_status_engine.Tpo -c -o mod_security2_la-msc_status_engine.lo `test -f 'msc_status_engine.c' || echo './'`msc_status_engine.c
/bin/sh ../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I.  -DLINUX=2 -D_REENTRANT -D_GNU_SOURCE -I/usr/include/libxml2   -I/usr/include/apr-1 -I/usr/include/apr-1 -I/usr/include/httpd -I/usr/include/libxml2 -DWITH_LUA -DWITH_PCRE_STUDY    -DREQUEST_EARLY  -DCACHE_LUA -DWITH_YAJL -I/usr/include/yajl  -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector --param=ssp-buffer-size=4 -m64 -mtune=generic  -I/usr/include/apr-1  -MT mod_security2_la-msc_crypt.lo -MD -MP -MF .deps/mod_security2_la-msc_crypt.Tpo -c -o mod_security2_la-msc_crypt.lo `test -f 'msc_crypt.c' || echo './'`msc_crypt.c
/bin/sh ../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I.  -DLINUX=2 -D_REENTRANT -D_GNU_SOURCE -I/usr/include/libxml2   -I/usr/include/apr-1 -I/usr/include/apr-1 -I/usr/include/httpd -I/usr/include/libxml2 -DWITH_LUA -DWITH_PCRE_STUDY    -DREQUEST_EARLY  -DCACHE_LUA -DWITH_YAJL -I/usr/include/yajl  -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector --param=ssp-buffer-size=4 -m64 -mtune=generic  -I/usr/include/apr-1  -MT mod_security2_la-msc_geo.lo -MD -MP -MF .deps/mod_security2_la-msc_geo.Tpo -c -o mod_security2_la-msc_geo.lo `test -f 'msc_geo.c' || echo './'`msc_geo.c
/bin/sh ../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I.  -DLINUX=2 -D_REENTRANT -D_GNU_SOURCE -I/usr/include/libxml2   -I/usr/include/apr-1 -I/usr/include/apr-1 -I/usr/include/httpd -I/usr/include/libxml2 -DWITH_LUA -DWITH_PCRE_STUDY    -DREQUEST_EARLY  -DCACHE_LUA -DWITH_YAJL -I/usr/include/yajl  -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector --param=ssp-buffer-size=4 -m64 -mtune=generic  -I/usr/include/apr-1  -MT mod_security2_la-msc_gsb.lo -MD -MP -MF .deps/mod_security2_la-msc_gsb.Tpo -c -o mod_security2_la-msc_gsb.lo `test -f 'msc_gsb.c' || echo './'`msc_gsb.c
/bin/sh ../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I.  -DLINUX=2 -D_REENTRANT -D_GNU_SOURCE -I/usr/include/libxml2   -I/usr/include/apr-1 -I/usr/include/apr-1 -I/usr/include/httpd -I/usr/include/libxml2 -DWITH_LUA -DWITH_PCRE_STUDY    -DREQUEST_EARLY  -DCACHE_LUA -DWITH_YAJL -I/usr/include/yajl  -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector --param=ssp-buffer-size=4 -m64 -mtune=generic  -I/usr/include/apr-1  -MT mod_security2_la-msc_json.lo -MD -MP -MF .deps/mod_security2_la-msc_json.Tpo -c -o mod_security2_la-msc_json.lo `test -f 'msc_json.c' || echo './'`msc_json.c
/bin/sh ../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I.  -DLINUX=2 -D_REENTRANT -D_GNU_SOURCE -I/usr/include/libxml2   -I/usr/include/apr-1 -I/usr/include/apr-1 -I/usr/include/httpd -I/usr/include/libxml2 -DWITH_LUA -DWITH_PCRE_STUDY    -DREQUEST_EARLY  -DCACHE_LUA -DWITH_YAJL -I/usr/include/yajl  -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector --param=ssp-buffer-size=4 -m64 -mtune=generic  -I/usr/include/apr-1  -MT mod_security2_la-msc_logging.lo -MD -MP -MF .deps/mod_security2_la-msc_logging.Tpo -c -o mod_security2_la-msc_logging.lo `test -f 'msc_logging.c' || echo './'`msc_logging.c
/bin/sh ../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I.  -DLINUX=2 -D_REENTRANT -D_GNU_SOURCE -I/usr/include/libxml2   -I/usr/include/apr-1 -I/usr/include/apr-1 -I/usr/include/httpd -I/usr/include/libxml2 -DWITH_LUA -DWITH_PCRE_STUDY    -DREQUEST_EARLY  -DCACHE_LUA -DWITH_YAJL -I/usr/include/yajl  -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector --param=ssp-buffer-size=4 -m64 -mtune=generic  -I/usr/include/apr-1  -MT mod_security2_la-msc_lua.lo -MD -MP -MF .deps/mod_security2_la-msc_lua.Tpo -c -o mod_security2_la-msc_lua.lo `test -f 'msc_lua.c' || echo './'`msc_lua.c
/bin/sh ../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I.  -DLINUX=2 -D_REENTRANT -D_GNU_SOURCE -I/usr/include/libxml2   -I/usr/include/apr-1 -I/usr/include/apr-1 -I/usr/include/httpd -I/usr/include/libxml2 -DWITH_LUA -DWITH_PCRE_STUDY    -DREQUEST_EARLY  -DCACHE_LUA -DWITH_YAJL -I/usr/include/yajl  -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector --param=ssp-buffer-size=4 -m64 -mtune=generic  -I/usr/include/apr-1  -MT mod_security2_la-msc_multipart.lo -MD -MP -MF .deps/mod_security2_la-msc_multipart.Tpo -c -o mod_security2_la-msc_multipart.lo `test -f 'msc_multipart.c' || echo './'`msc_multipart.c
/bin/sh ../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I.  -DLINUX=2 -D_REENTRANT -D_GNU_SOURCE -I/usr/include/libxml2   -I/usr/include/apr-1 -I/usr/include/apr-1 -I/usr/include/httpd -I/usr/include/libxml2 -DWITH_LUA -DWITH_PCRE_STUDY    -DREQUEST_EARLY  -DCACHE_LUA -DWITH_YAJL -I/usr/include/yajl  -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector --param=ssp-buffer-size=4 -m64 -mtune=generic  -I/usr/include/apr-1  -MT mod_security2_la-msc_parsers.lo -MD -MP -MF .deps/mod_security2_la-msc_parsers.Tpo -c -o mod_security2_la-msc_parsers.lo `test -f 'msc_parsers.c' || echo './'`msc_parsers.c
/bin/sh ../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I.  -DLINUX=2 -D_REENTRANT -D_GNU_SOURCE -I/usr/include/libxml2   -I/usr/include/apr-1 -I/usr/include/apr-1 -I/usr/include/httpd -I/usr/include/libxml2 -DWITH_LUA -DWITH_PCRE_STUDY    -DREQUEST_EARLY  -DCACHE_LUA -DWITH_YAJL -I/usr/include/yajl  -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector --param=ssp-buffer-size=4 -m64 -mtune=generic  -I/usr/include/apr-1  -MT mod_security2_la-msc_pcre.lo -MD -MP -MF .deps/mod_security2_la-msc_pcre.Tpo -c -o mod_security2_la-msc_pcre.lo `test -f 'msc_pcre.c' || echo './'`msc_pcre.c
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -DLINUX=2 -D_REENTRANT -D_GNU_SOURCE -I/usr/include/libxml2 -I/usr/include/apr-1 -I/usr/include/apr-1 -I/usr/include/httpd -I/usr/include/libxml2 -DWITH_LUA -DWITH_PCRE_STUDY -DREQUEST_EARLY -DCACHE_LUA -DWITH_YAJL -I/usr/include/yajl -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector --param=ssp-buffer-size=4 -m64 -mtune=generic -I/usr/include/apr-1 -MT mod_security2_la-acmp.lo -MD -MP -MF .deps/mod_security2_la-acmp.Tpo -c acmp.c  -fPIC -DPIC -o .libs/mod_security2_la-acmp.o
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -DLINUX=2 -D_REENTRANT -D_GNU_SOURCE -I/usr/include/libxml2 -I/usr/include/apr-1 -I/usr/include/apr-1 -I/usr/include/httpd -I/usr/include/libxml2 -DWITH_LUA -DWITH_PCRE_STUDY -DREQUEST_EARLY -DCACHE_LUA -DWITH_YAJL -I/usr/include/yajl -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector --param=ssp-buffer-size=4 -m64 -mtune=generic -I/usr/include/apr-1 -MT mod_security2_la-apache2_config.lo -MD -MP -MF .deps/mod_security2_la-apache2_config.Tpo -c apache2_config.c  -fPIC -DPIC -o .libs/mod_security2_la-apache2_config.o
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -DLINUX=2 -D_REENTRANT -D_GNU_SOURCE -I/usr/include/libxml2 -I/usr/include/apr-1 -I/usr/include/apr-1 -I/usr/include/httpd -I/usr/include/libxml2 -DWITH_LUA -DWITH_PCRE_STUDY -DREQUEST_EARLY -DCACHE_LUA -DWITH_YAJL -I/usr/include/yajl -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector --param=ssp-buffer-size=4 -m64 -mtune=generic -I/usr/include/apr-1 -MT mod_security2_la-apache2_io.lo -MD -MP -MF .deps/mod_security2_la-apache2_io.Tpo -c apache2_io.c  -fPIC -DPIC -o .libs/mod_security2_la-apache2_io.o
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -DLINUX=2 -D_REENTRANT -D_GNU_SOURCE -I/usr/include/libxml2 -I/usr/include/apr-1 -I/usr/include/apr-1 -I/usr/include/httpd -I/usr/include/libxml2 -DWITH_LUA -DWITH_PCRE_STUDY -DREQUEST_EARLY -DCACHE_LUA -DWITH_YAJL -I/usr/include/yajl -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector --param=ssp-buffer-size=4 -m64 -mtune=generic -I/usr/include/apr-1 -MT mod_security2_la-apache2_util.lo -MD -MP -MF .deps/mod_security2_la-apache2_util.Tpo -c apache2_util.c  -fPIC -DPIC -o .libs/mod_security2_la-apache2_util.o
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -DLINUX=2 -D_REENTRANT -D_GNU_SOURCE -I/usr/include/libxml2 -I/usr/include/apr-1 -I/usr/include/apr-1 -I/usr/include/httpd -I/usr/include/libxml2 -DWITH_LUA -DWITH_PCRE_STUDY -DREQUEST_EARLY -DCACHE_LUA -DWITH_YAJL -I/usr/include/yajl -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector --param=ssp-buffer-size=4 -m64 -mtune=generic -I/usr/include/apr-1 -MT mod_security2_la-mod_security2.lo -MD -MP -MF .deps/mod_security2_la-mod_security2.Tpo -c mod_security2.c  -fPIC -DPIC -o .libs/mod_security2_la-mod_security2.o
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -DLINUX=2 -D_REENTRANT -D_GNU_SOURCE -I/usr/include/libxml2 -I/usr/include/apr-1 -I/usr/include/apr-1 -I/usr/include/httpd -I/usr/include/libxml2 -DWITH_LUA -DWITH_PCRE_STUDY -DREQUEST_EARLY -DCACHE_LUA -DWITH_YAJL -I/usr/include/yajl -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector --param=ssp-buffer-size=4 -m64 -mtune=generic -I/usr/include/apr-1 -MT mod_security2_la-modsecurity.lo -MD -MP -MF .deps/mod_security2_la-modsecurity.Tpo -c modsecurity.c  -fPIC -DPIC -o .libs/mod_security2_la-modsecurity.o
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -DLINUX=2 -D_REENTRANT -D_GNU_SOURCE -I/usr/include/libxml2 -I/usr/include/apr-1 -I/usr/include/apr-1 -I/usr/include/httpd -I/usr/include/libxml2 -DWITH_LUA -DWITH_PCRE_STUDY -DREQUEST_EARLY -DCACHE_LUA -DWITH_YAJL -I/usr/include/yajl -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector --param=ssp-buffer-size=4 -m64 -mtune=generic -I/usr/include/apr-1 -MT mod_security2_la-msc_status_engine.lo -MD -MP -MF .deps/mod_security2_la-msc_status_engine.Tpo -c msc_status_engine.c  -fPIC -DPIC -o .libs/mod_security2_la-msc_status_engine.o
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -DLINUX=2 -D_REENTRANT -D_GNU_SOURCE -I/usr/include/libxml2 -I/usr/include/apr-1 -I/usr/include/apr-1 -I/usr/include/httpd -I/usr/include/libxml2 -DWITH_LUA -DWITH_PCRE_STUDY -DREQUEST_EARLY -DCACHE_LUA -DWITH_YAJL -I/usr/include/yajl -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector --param=ssp-buffer-size=4 -m64 -mtune=generic -I/usr/include/apr-1 -MT mod_security2_la-msc_geo.lo -MD -MP -MF .deps/mod_security2_la-msc_geo.Tpo -c msc_geo.c  -fPIC -DPIC -o .libs/mod_security2_la-msc_geo.o
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -DLINUX=2 -D_REENTRANT -D_GNU_SOURCE -I/usr/include/libxml2 -I/usr/include/apr-1 -I/usr/include/apr-1 -I/usr/include/httpd -I/usr/include/libxml2 -DWITH_LUA -DWITH_PCRE_STUDY -DREQUEST_EARLY -DCACHE_LUA -DWITH_YAJL -I/usr/include/yajl -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector --param=ssp-buffer-size=4 -m64 -mtune=generic -I/usr/include/apr-1 -MT mod_security2_la-msc_gsb.lo -MD -MP -MF .deps/mod_security2_la-msc_gsb.Tpo -c msc_gsb.c  -fPIC -DPIC -o .libs/mod_security2_la-msc_gsb.o
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -DLINUX=2 -D_REENTRANT -D_GNU_SOURCE -I/usr/include/libxml2 -I/usr/include/apr-1 -I/usr/include/apr-1 -I/usr/include/httpd -I/usr/include/libxml2 -DWITH_LUA -DWITH_PCRE_STUDY -DREQUEST_EARLY -DCACHE_LUA -DWITH_YAJL -I/usr/include/yajl -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector --param=ssp-buffer-size=4 -m64 -mtune=generic -I/usr/include/apr-1 -MT mod_security2_la-msc_crypt.lo -MD -MP -MF .deps/mod_security2_la-msc_crypt.Tpo -c msc_crypt.c  -fPIC -DPIC -o .libs/mod_security2_la-msc_crypt.o
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -DLINUX=2 -D_REENTRANT -D_GNU_SOURCE -I/usr/include/libxml2 -I/usr/include/apr-1 -I/usr/include/apr-1 -I/usr/include/httpd -I/usr/include/libxml2 -DWITH_LUA -DWITH_PCRE_STUDY -DREQUEST_EARLY -DCACHE_LUA -DWITH_YAJL -I/usr/include/yajl -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector --param=ssp-buffer-size=4 -m64 -mtune=generic -I/usr/include/apr-1 -MT mod_security2_la-msc_json.lo -MD -MP -MF .deps/mod_security2_la-msc_json.Tpo -c msc_json.c  -fPIC -DPIC -o .libs/mod_security2_la-msc_json.o
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -DLINUX=2 -D_REENTRANT -D_GNU_SOURCE -I/usr/include/libxml2 -I/usr/include/apr-1 -I/usr/include/apr-1 -I/usr/include/httpd -I/usr/include/libxml2 -DWITH_LUA -DWITH_PCRE_STUDY -DREQUEST_EARLY -DCACHE_LUA -DWITH_YAJL -I/usr/include/yajl -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector --param=ssp-buffer-size=4 -m64 -mtune=generic -I/usr/include/apr-1 -MT mod_security2_la-msc_lua.lo -MD -MP -MF .deps/mod_security2_la-msc_lua.Tpo -c msc_lua.c  -fPIC -DPIC -o .libs/mod_security2_la-msc_lua.o
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -DLINUX=2 -D_REENTRANT -D_GNU_SOURCE -I/usr/include/libxml2 -I/usr/include/apr-1 -I/usr/include/apr-1 -I/usr/include/httpd -I/usr/include/libxml2 -DWITH_LUA -DWITH_PCRE_STUDY -DREQUEST_EARLY -DCACHE_LUA -DWITH_YAJL -I/usr/include/yajl -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector --param=ssp-buffer-size=4 -m64 -mtune=generic -I/usr/include/apr-1 -MT mod_security2_la-msc_multipart.lo -MD -MP -MF .deps/mod_security2_la-msc_multipart.Tpo -c msc_multipart.c  -fPIC -DPIC -o .libs/mod_security2_la-msc_multipart.o
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -DLINUX=2 -D_REENTRANT -D_GNU_SOURCE -I/usr/include/libxml2 -I/usr/include/apr-1 -I/usr/include/apr-1 -I/usr/include/httpd -I/usr/include/libxml2 -DWITH_LUA -DWITH_PCRE_STUDY -DREQUEST_EARLY -DCACHE_LUA -DWITH_YAJL -I/usr/include/yajl -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector --param=ssp-buffer-size=4 -m64 -mtune=generic -I/usr/include/apr-1 -MT mod_security2_la-msc_logging.lo -MD -MP -MF .deps/mod_security2_la-msc_logging.Tpo -c msc_logging.c  -fPIC -DPIC -o .libs/mod_security2_la-msc_logging.o
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -DLINUX=2 -D_REENTRANT -D_GNU_SOURCE -I/usr/include/libxml2 -I/usr/include/apr-1 -I/usr/include/apr-1 -I/usr/include/httpd -I/usr/include/libxml2 -DWITH_LUA -DWITH_PCRE_STUDY -DREQUEST_EARLY -DCACHE_LUA -DWITH_YAJL -I/usr/include/yajl -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector --param=ssp-buffer-size=4 -m64 -mtune=generic -I/usr/include/apr-1 -MT mod_security2_la-msc_parsers.lo -MD -MP -MF .deps/mod_security2_la-msc_parsers.Tpo -c msc_parsers.c  -fPIC -DPIC -o .libs/mod_security2_la-msc_parsers.o
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -DLINUX=2 -D_REENTRANT -D_GNU_SOURCE -I/usr/include/libxml2 -I/usr/include/apr-1 -I/usr/include/apr-1 -I/usr/include/httpd -I/usr/include/libxml2 -DWITH_LUA -DWITH_PCRE_STUDY -DREQUEST_EARLY -DCACHE_LUA -DWITH_YAJL -I/usr/include/yajl -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector --param=ssp-buffer-size=4 -m64 -mtune=generic -I/usr/include/apr-1 -MT mod_security2_la-msc_pcre.lo -MD -MP -MF .deps/mod_security2_la-msc_pcre.Tpo -c msc_pcre.c  -fPIC -DPIC -o .libs/mod_security2_la-msc_pcre.o
acmp.c:258: warning: 'acmp_clone_node_no_state' defined but not used
apache2_util.c: In function 'internal_log_ex':
apache2_util.c:204: warning: unused variable 'str'
apache2_util.c:203: warning: unused variable 'saved'
apache2_util.c:202: warning: unused variable 'parse_remote'
apache2_util.c:201: warning: unused variable 'remote'
apache2_config.c: In function 'add_rule':
apache2_config.c:771: warning: unused variable 'offset'
msc_status_engine.c: In function 'msc_status_engine_mac_address':
msc_status_engine.c:198: warning: implicit declaration of function 'ioctl'
msc_status_engine.c:221: warning: pointer targets in passing argument 1 of 'apr_snprintf' differ in signedness
/usr/include/apr-1/apr_strings.h:261: note: expected 'char *' but argument is of type 'unsigned char *'
msc_status_engine.c:209: warning: unused variable 'i'
msc_status_engine.c: In function 'msc_status_engine_unique_id':
msc_status_engine.c:314: warning: pointer targets in passing argument 1 of 'strlen' differ in signedness
/usr/include/string.h:399: note: expected 'const char *' but argument is of type 'unsigned char *'
msc_status_engine.c:314: warning: pointer targets in passing argument 2 of 'apr_sha1_update' differ in signedness
/usr/include/apr-1/apr_sha1.h:96: note: expected 'const char *' but argument is of type 'unsigned char *'
msc_status_engine.c:319: warning: pointer targets in passing argument 1 of 'sprintf' differ in signedness
/usr/include/bits/stdio2.h:32: note: expected 'char * __restrict__' but argument is of type 'unsigned char *'
msc_status_engine.c: In function 'msc_beacon_string':
msc_status_engine.c:363: warning: the address of 'pcre' will always evaluate as 'true'
msc_status_engine.c:376: warning: pointer targets in passing argument 1 of 'msc_status_engine_unique_id' differ in signedness
msc_status_engine.c:280: note: expected 'unsigned char *' but argument is of type 'char *'
msc_json.c: In function 'yajl_map_key':
msc_json.c:85: warning: pointer targets in passing argument 2 of 'apr_pstrndup' differ in signedness
/usr/include/apr-1/apr_strings.h:121: note: expected 'const char *' but argument is of type 'const unsigned char *'
msc_json.c:85: warning: pointer targets in assignment differ in signedness
msc_status_engine.c: In function 'msc_status_engine_call':
msc_status_engine.c:478: warning: implicit declaration of function 'gethostbyname'
msc_json.c: In function 'yajl_string':
msc_json.c:134: warning: pointer targets in passing argument 2 of 'json_add_argument' differ in signedness
msc_json.c:19: note: expected 'const char *' but argument is of type 'const unsigned char *'
msc_json.c: In function 'yajl_start_map':
msc_json.c:169: warning: pointer targets in assignment differ in signedness
msc_json.c:172: warning: pointer targets in passing argument 2 of 'apr_pstrdup' differ in signedness
/usr/include/apr-1/apr_strings.h:95: note: expected 'const char *' but argument is of type 'unsigned char *'
msc_json.c:172: warning: pointer targets in assignment differ in signedness
msc_json.c: In function 'yajl_end_map':
msc_json.c:201: warning: pointer targets in passing argument 1 of 'strrchr' differ in signedness
/usr/include/string.h:262: note: expected 'const char *' but argument is of type 'unsigned char *'
msc_json.c:201: warning: pointer targets in assignment differ in signedness
msc_json.c:205: warning: pointer targets in passing argument 2 of 'apr_pstrmemdup' differ in signedness
/usr/include/apr-1/apr_strings.h:109: note: expected 'const char *' but argument is of type 'unsigned char *'
msc_json.c:205: warning: pointer targets in assignment differ in signedness
msc_json.c:206: warning: pointer targets in assignment differ in signedness
msc_json.c: In function 'json_process_chunk':
msc_json.c:278: warning: pointer targets in passing argument 2 of 'yajl_parse' differ in signedness
/usr/include/yajl/yajl_parse.h:176: note: expected 'const unsigned char *' but argument is of type 'const char *'
msc_json.c:281: warning: pointer targets in passing argument 3 of 'yajl_get_error' differ in signedness
/usr/include/yajl/yajl_parse.h:201: note: expected 'const unsigned char *' but argument is of type 'const char *'
msc_json.c:281: warning: pointer targets in assignment differ in signedness
msc_json.c: In function 'json_complete':
msc_json.c:301: warning: pointer targets in assignment differ in signedness
msc_json.c:292: warning: unused variable 'json_data'
msc_crypt.c: In function 'normalize_path':
msc_crypt.c:69: warning: unused variable 'bytes'
mod_security2.c: In function 'hook_connection_early':
mod_security2.c:1455: error: 'sb_handle' undeclared (first use in this function)
mod_security2.c:1455: error: (Each undeclared identifier is reported only once
mod_security2.c:1455: error: for each function it appears in.)
mod_security2.c:1455: error: 'sbh' undeclared (first use in this function)
msc_crypt.c: In function 'getkey':
msc_crypt.c:154: warning: pointer targets in passing argument 1 of 'apr_generate_random_bytes' differ in signedness
/usr/include/apr-1/apr_general.h:231: note: expected 'unsigned char *' but argument is of type 'char *'
msc_crypt.c:161: warning: pointer targets in passing argument 1 of 'apr_generate_random_bytes' differ in signedness
/usr/include/apr-1/apr_general.h:231: note: expected 'unsigned char *' but argument is of type 'char *'
mod_security2.c:1487: error: 'APLOG_TRACE3' undeclared (first use in this function)
apache2_config.c: In function 'cmd_remote_rules':
apache2_config.c:2278: warning: unused variable 'error_msg'
msc_crypt.c: In function 'inject_hashed_response_body':
msc_crypt.c:1169: warning: format '%d' expects type 'int', but argument 4 has type 'apr_size_t'
msc_crypt.c:1199: warning: format '%d' expects type 'int', but argument 4 has type 'apr_size_t'
make[2]: *** [mod_security2_la-mod_security2.lo] Error 1
make[2]: *** Waiting for unfinished jobs....
msc_logging.c: In function 'sec_audit_logger_json':
msc_logging.c:1117: warning: unused variable 'now'
msc_logging.c:1352: warning: pointer targets in passing argument 2 of 'yajl_gen_string' differ in signedness
/usr/include/yajl/yajl_gen.h:131: note: expected 'const unsigned char *' but argument is of type 'char * const'
msc_logging.c:1430: warning: pointer targets in passing argument 2 of 'sec_auditlog_write' differ in signedness
msc_logging.c:37: note: expected 'const char *' but argument is of type 'const unsigned char *'
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -DLINUX=2 -D_REENTRANT -D_GNU_SOURCE -I/usr/include/libxml2 -I/usr/include/apr-1 -I/usr/include/apr-1 -I/usr/include/httpd -I/usr/include/libxml2 -DWITH_LUA -DWITH_PCRE_STUDY -DREQUEST_EARLY -DCACHE_LUA -DWITH_YAJL -I/usr/include/yajl -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector --param=ssp-buffer-size=4 -m64 -mtune=generic -I/usr/include/apr-1 -MT mod_security2_la-msc_pcre.lo -MD -MP -MF .deps/mod_security2_la-msc_pcre.Tpo -c msc_pcre.c -o mod_security2_la-msc_pcre.o >/dev/null 2>&1
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -DLINUX=2 -D_REENTRANT -D_GNU_SOURCE -I/usr/include/libxml2 -I/usr/include/apr-1 -I/usr/include/apr-1 -I/usr/include/httpd -I/usr/include/libxml2 -DWITH_LUA -DWITH_PCRE_STUDY -DREQUEST_EARLY -DCACHE_LUA -DWITH_YAJL -I/usr/include/yajl -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector --param=ssp-buffer-size=4 -m64 -mtune=generic -I/usr/include/apr-1 -MT mod_security2_la-msc_gsb.lo -MD -MP -MF .deps/mod_security2_la-msc_gsb.Tpo -c msc_gsb.c -o mod_security2_la-msc_gsb.o >/dev/null 2>&1
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -DLINUX=2 -D_REENTRANT -D_GNU_SOURCE -I/usr/include/libxml2 -I/usr/include/apr-1 -I/usr/include/apr-1 -I/usr/include/httpd -I/usr/include/libxml2 -DWITH_LUA -DWITH_PCRE_STUDY -DREQUEST_EARLY -DCACHE_LUA -DWITH_YAJL -I/usr/include/yajl -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector --param=ssp-buffer-size=4 -m64 -mtune=generic -I/usr/include/apr-1 -MT mod_security2_la-acmp.lo -MD -MP -MF .deps/mod_security2_la-acmp.Tpo -c acmp.c -o mod_security2_la-acmp.o >/dev/null 2>&1
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -DLINUX=2 -D_REENTRANT -D_GNU_SOURCE -I/usr/include/libxml2 -I/usr/include/apr-1 -I/usr/include/apr-1 -I/usr/include/httpd -I/usr/include/libxml2 -DWITH_LUA -DWITH_PCRE_STUDY -DREQUEST_EARLY -DCACHE_LUA -DWITH_YAJL -I/usr/include/yajl -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector --param=ssp-buffer-size=4 -m64 -mtune=generic -I/usr/include/apr-1 -MT mod_security2_la-msc_json.lo -MD -MP -MF .deps/mod_security2_la-msc_json.Tpo -c msc_json.c -o mod_security2_la-msc_json.o >/dev/null 2>&1
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -DLINUX=2 -D_REENTRANT -D_GNU_SOURCE -I/usr/include/libxml2 -I/usr/include/apr-1 -I/usr/include/apr-1 -I/usr/include/httpd -I/usr/include/libxml2 -DWITH_LUA -DWITH_PCRE_STUDY -DREQUEST_EARLY -DCACHE_LUA -DWITH_YAJL -I/usr/include/yajl -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector --param=ssp-buffer-size=4 -m64 -mtune=generic -I/usr/include/apr-1 -MT mod_security2_la-msc_status_engine.lo -MD -MP -MF .deps/mod_security2_la-msc_status_engine.Tpo -c msc_status_engine.c -o mod_security2_la-msc_status_engine.o >/dev/null 2>&1
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -DLINUX=2 -D_REENTRANT -D_GNU_SOURCE -I/usr/include/libxml2 -I/usr/include/apr-1 -I/usr/include/apr-1 -I/usr/include/httpd -I/usr/include/libxml2 -DWITH_LUA -DWITH_PCRE_STUDY -DREQUEST_EARLY -DCACHE_LUA -DWITH_YAJL -I/usr/include/yajl -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector --param=ssp-buffer-size=4 -m64 -mtune=generic -I/usr/include/apr-1 -MT mod_security2_la-apache2_util.lo -MD -MP -MF .deps/mod_security2_la-apache2_util.Tpo -c apache2_util.c -o mod_security2_la-apache2_util.o >/dev/null 2>&1
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -DLINUX=2 -D_REENTRANT -D_GNU_SOURCE -I/usr/include/libxml2 -I/usr/include/apr-1 -I/usr/include/apr-1 -I/usr/include/httpd -I/usr/include/libxml2 -DWITH_LUA -DWITH_PCRE_STUDY -DREQUEST_EARLY -DCACHE_LUA -DWITH_YAJL -I/usr/include/yajl -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector --param=ssp-buffer-size=4 -m64 -mtune=generic -I/usr/include/apr-1 -MT mod_security2_la-msc_lua.lo -MD -MP -MF .deps/mod_security2_la-msc_lua.Tpo -c msc_lua.c -o mod_security2_la-msc_lua.o >/dev/null 2>&1
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -DLINUX=2 -D_REENTRANT -D_GNU_SOURCE -I/usr/include/libxml2 -I/usr/include/apr-1 -I/usr/include/apr-1 -I/usr/include/httpd -I/usr/include/libxml2 -DWITH_LUA -DWITH_PCRE_STUDY -DREQUEST_EARLY -DCACHE_LUA -DWITH_YAJL -I/usr/include/yajl -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector --param=ssp-buffer-size=4 -m64 -mtune=generic -I/usr/include/apr-1 -MT mod_security2_la-msc_geo.lo -MD -MP -MF .deps/mod_security2_la-msc_geo.Tpo -c msc_geo.c -o mod_security2_la-msc_geo.o >/dev/null 2>&1
msc_crypt.c: In function 'init_response_body_html_parser':
msc_crypt.c:244: warning: 'sep' may be used uninitialized in this function
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -DLINUX=2 -D_REENTRANT -D_GNU_SOURCE -I/usr/include/libxml2 -I/usr/include/apr-1 -I/usr/include/apr-1 -I/usr/include/httpd -I/usr/include/libxml2 -DWITH_LUA -DWITH_PCRE_STUDY -DREQUEST_EARLY -DCACHE_LUA -DWITH_YAJL -I/usr/include/yajl -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector --param=ssp-buffer-size=4 -m64 -mtune=generic -I/usr/include/apr-1 -MT mod_security2_la-msc_parsers.lo -MD -MP -MF .deps/mod_security2_la-msc_parsers.Tpo -c msc_parsers.c -o mod_security2_la-msc_parsers.o >/dev/null 2>&1
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -DLINUX=2 -D_REENTRANT -D_GNU_SOURCE -I/usr/include/libxml2 -I/usr/include/apr-1 -I/usr/include/apr-1 -I/usr/include/httpd -I/usr/include/libxml2 -DWITH_LUA -DWITH_PCRE_STUDY -DREQUEST_EARLY -DCACHE_LUA -DWITH_YAJL -I/usr/include/yajl -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector --param=ssp-buffer-size=4 -m64 -mtune=generic -I/usr/include/apr-1 -MT mod_security2_la-modsecurity.lo -MD -MP -MF .deps/mod_security2_la-modsecurity.Tpo -c modsecurity.c -o mod_security2_la-modsecurity.o >/dev/null 2>&1
mv -f .deps/mod_security2_la-msc_pcre.Tpo .deps/mod_security2_la-msc_pcre.Plo
mv -f .deps/mod_security2_la-msc_gsb.Tpo .deps/mod_security2_la-msc_gsb.Plo
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -DLINUX=2 -D_REENTRANT -D_GNU_SOURCE -I/usr/include/libxml2 -I/usr/include/apr-1 -I/usr/include/apr-1 -I/usr/include/httpd -I/usr/include/libxml2 -DWITH_LUA -DWITH_PCRE_STUDY -DREQUEST_EARLY -DCACHE_LUA -DWITH_YAJL -I/usr/include/yajl -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector --param=ssp-buffer-size=4 -m64 -mtune=generic -I/usr/include/apr-1 -MT mod_security2_la-apache2_io.lo -MD -MP -MF .deps/mod_security2_la-apache2_io.Tpo -c apache2_io.c -o mod_security2_la-apache2_io.o >/dev/null 2>&1
mv -f .deps/mod_security2_la-acmp.Tpo .deps/mod_security2_la-acmp.Plo
mv -f .deps/mod_security2_la-msc_json.Tpo .deps/mod_security2_la-msc_json.Plo
mv -f .deps/mod_security2_la-msc_status_engine.Tpo .deps/mod_security2_la-msc_status_engine.Plo
mv -f .deps/mod_security2_la-msc_lua.Tpo .deps/mod_security2_la-msc_lua.Plo
mv -f .deps/mod_security2_la-apache2_util.Tpo .deps/mod_security2_la-apache2_util.Plo
mv -f .deps/mod_security2_la-msc_geo.Tpo .deps/mod_security2_la-msc_geo.Plo
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -DLINUX=2 -D_REENTRANT -D_GNU_SOURCE -I/usr/include/libxml2 -I/usr/include/apr-1 -I/usr/include/apr-1 -I/usr/include/httpd -I/usr/include/libxml2 -DWITH_LUA -DWITH_PCRE_STUDY -DREQUEST_EARLY -DCACHE_LUA -DWITH_YAJL -I/usr/include/yajl -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector --param=ssp-buffer-size=4 -m64 -mtune=generic -I/usr/include/apr-1 -MT mod_security2_la-msc_crypt.lo -MD -MP -MF .deps/mod_security2_la-msc_crypt.Tpo -c msc_crypt.c -o mod_security2_la-msc_crypt.o >/dev/null 2>&1
mv -f .deps/mod_security2_la-msc_parsers.Tpo .deps/mod_security2_la-msc_parsers.Plo
mv -f .deps/mod_security2_la-modsecurity.Tpo .deps/mod_security2_la-modsecurity.Plo
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -DLINUX=2 -D_REENTRANT -D_GNU_SOURCE -I/usr/include/libxml2 -I/usr/include/apr-1 -I/usr/include/apr-1 -I/usr/include/httpd -I/usr/include/libxml2 -DWITH_LUA -DWITH_PCRE_STUDY -DREQUEST_EARLY -DCACHE_LUA -DWITH_YAJL -I/usr/include/yajl -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector --param=ssp-buffer-size=4 -m64 -mtune=generic -I/usr/include/apr-1 -MT mod_security2_la-msc_multipart.lo -MD -MP -MF .deps/mod_security2_la-msc_multipart.Tpo -c msc_multipart.c -o mod_security2_la-msc_multipart.o >/dev/null 2>&1
mv -f .deps/mod_security2_la-apache2_io.Tpo .deps/mod_security2_la-apache2_io.Plo
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -DLINUX=2 -D_REENTRANT -D_GNU_SOURCE -I/usr/include/libxml2 -I/usr/include/apr-1 -I/usr/include/apr-1 -I/usr/include/httpd -I/usr/include/libxml2 -DWITH_LUA -DWITH_PCRE_STUDY -DREQUEST_EARLY -DCACHE_LUA -DWITH_YAJL -I/usr/include/yajl -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector --param=ssp-buffer-size=4 -m64 -mtune=generic -I/usr/include/apr-1 -MT mod_security2_la-apache2_config.lo -MD -MP -MF .deps/mod_security2_la-apache2_config.Tpo -c apache2_config.c -o mod_security2_la-apache2_config.o >/dev/null 2>&1
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -DLINUX=2 -D_REENTRANT -D_GNU_SOURCE -I/usr/include/libxml2 -I/usr/include/apr-1 -I/usr/include/apr-1 -I/usr/include/httpd -I/usr/include/libxml2 -DWITH_LUA -DWITH_PCRE_STUDY -DREQUEST_EARLY -DCACHE_LUA -DWITH_YAJL -I/usr/include/yajl -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector --param=ssp-buffer-size=4 -m64 -mtune=generic -I/usr/include/apr-1 -MT mod_security2_la-msc_logging.lo -MD -MP -MF .deps/mod_security2_la-msc_logging.Tpo -c msc_logging.c -o mod_security2_la-msc_logging.o >/dev/null 2>&1
mv -f .deps/mod_security2_la-msc_crypt.Tpo .deps/mod_security2_la-msc_crypt.Plo
mv -f .deps/mod_security2_la-msc_multipart.Tpo .deps/mod_security2_la-msc_multipart.Plo
mv -f .deps/mod_security2_la-apache2_config.Tpo .deps/mod_security2_la-apache2_config.Plo
mv -f .deps/mod_security2_la-msc_logging.Tpo .deps/mod_security2_la-msc_logging.Plo
make[2]: Leaving directory `/builddir/build/BUILD/ModSecurity-master/apache2'
make[1]: *** [all] Error 2
make[1]: Leaving directory `/builddir/build/BUILD/ModSecurity-master/apache2'
make: *** [all-recursive] Error 1
error: Bad exit status from /var/tmp/rpm-tmp.cduVKL (%build)

````

**Comment by zimmerle** [comment]
Hi @emphazer,

Sorry for the trouble. that was a consequence of the merge #1340. Already committed a fix here -
https://github.com/SpiderLabs/ModSecurity/tree/v2/dev/issue_1407

Running in our buildbots before merge it to ```v2/master```. Do you mind to test this branch?

**Comment by zimmerle** [comment]
Our apache 2.2 build agent is down for some reason -
http://modsec.zimmerle.org/buildbot/buildslaves/Linux64-apache22

Checking to see if i can bring it to life again.

**Comment by zimmerle** [comment]
buildbot agent for apache 2.2 is working again.

**Comment by emphazer** [comment]
i receive following error with https://github.com/SpiderLabs/ModSecurity/tree/v2/dev/issue_1407
````
mod_security2.c:1495: error: 'APLOG_TRACE3' undeclared (first use in this function)
mod_security2.c:1495: error: (Each undeclared identifier is reported only once
mod_security2.c:1495: error: for each function it appears in.)
````

**Comment by joshgibs** [comment]
I still have the same compile error on OEL 6 using apache 2.2.

mod_security2.c:1495: error: 'APLOG_TRACE3' undeclared (first use in this function)

**Comment by zimmerle** [comment]
Fix merged into the master branch [master and v2/master].

da995bb636eab4b0a0e51df53605dd56e3c8c1b5

**Comment by joshgibs** [comment]
This solved my issue. Thank you!
